index.bs

```diff
@@ -3384,6 +3384,21 @@ To <dfn>locate nodes using inner text</dfn> with given |context nodes|,
 
 1. For each |context node| in |context nodes|:
 
+  1. If |context node| implements {{Document}} or {{DocumentFragment}}:
+
+    1. Let |child nodes| be an empty [=/list=].
+
+    1. For each node |child| in the <a spec=dom>children</a> of |context node|.
```

Comment:
Maybe use node's [`childNodes`](https://dom.spec.whatwg.org/#dom-node-childnodes) instead, as we are not interested in other types of children?

Reply:
Related issue in serialization: https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/705.html#issue-4a5a93fc

Reply:
Can be out of scope of the PR of course

Reply:
This is pre-existing text, so I am not sure. Do you want to file an issue?

Reply:
It would useful to know how exactly childNodes and children are different https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/705.html#issue-4a5a93fc ? here we link not to the DOM property though, but to the concept of children nodes so it looks fine to me in this place.